vagrant/provisioning/roles/arkcase-file-utils/tasks/main.yml

```diff
@@ -5,3 +5,59 @@
       name: arkcase-file-utils
       version: "{{ arkcase_file_utils_version | default(arkcase_version) }}"
       java_opts: "-Dserver.port=9068 -Dspring.cloud.config.uri=https://{{ config_server_host | default(internal_host) }}/config -Dspring.profiles.include=server,runtime -Djava.net.preferIPv4Stack=true -Djavax.net.ssl.trustStorePassword={{ java_trust_store_pass }} -Djavax.net.ssl.keyStore={{ java_key_store }} -Djavax.net.ssl.trustStore={{ java_trust_store}} -Djavax.net.ssl.keyStorePassword={{ java_key_store_pass }}"
+
+- name: Check Open Office installed
+  become: yes
+  stat:
+    path: "/opt/openoffice4/program/soffice"
+  register: openoffice_bin 
+
+- name: Check Open Office version
+  become: yes
+  command: "/opt/openoffice4/program/soffice -h  2>&1 | head -n1"
+  register: openoffice_old_version
```

Comment:
You might want the `when: openoffice_bin.stat.exists` guard here too, for clarity, even though on Ubuntu the exit code is 0 even when the executable doesn't exist (so the play won't fail).